### DISCUSSION TITLE
How do I to create a visible table of content for my kindle device on docusaurus?

### DISCUSSION THREAD

**Discussion by adonmez04** [facebook/docusaurus#8362] [Q&A] [answered]
Hi everyone,

I'm new here, and I'm trying to build my personal documentation website using docusaurus. 
I wonder how to add a table of content to my document for my kindle device... 
I used TOCInline and I have a table of content on my local website, it's really good, 
but when I send it to my kindle I can't see the same table of contents, I can't see anything. There is no the table of content on my kindle. 
Also when I print my docusaurus webpage, I can't see my table of content. 
What should I do? 
How do I to create a visible table of content for my kindle device on docusaurus?

Thank you so much...

Note: I've added some screenshots.

[I have a table of content on my local website.](https://i.imgur.com/qU9Hmcg.png)
[I cannot see the table of content on my kindle device.](https://i.imgur.com/j6tjCTw.png)

When I send it to my kindle device, I cannot see my table of content. 
I have to convert my local website to EPUB format to send to my kindle.
Here are some details about using [TOCInline.](https://docusaurus.io/docs/next/markdown-features/toc)

**Comment by lkzwc** [comment]
md can not realize?

**Reply by adonmez04** [reply]
Sorry, it was my fault...
I added some screenshots about my problem. 
Can you check my question again please?

**Comment by slorber** [comment, marked as answer]
I consider it's a bug in Infima, reported here: https://github.com/facebookincubator/infima/issues/278

You can probably revert this CSS rule locally using a custom class/selector:

```css
.table-of-contents {
  @media print {
    display: none;
  }
}
```

**Reply by adonmez04** [reply]
Thank you for your response. TOCInline is very useful, I hope this bug gets fixed.
